keystone.ts

```diff
@@ -16,7 +18,14 @@ export default withSharedAuth(
       enableLogging: true,
       useMigrations: true,
       prismaPreviewFeatures: ['fullTextSearch'],
+      async onConnect() {
+        await redisClient.connect()
+      },
     },
+    // @ts-expect-error We use a custom SharedSessionStrategy that does not
+    // include a `start` method. This makes the TS gods angry,
```

Comment:
nit-pick(non-blocking): just to avoid comments that might be taken the wrong way.

```suggestion
    // include a `start` method. This makes the TS compiler angry,
```